lib/SILGen/SILGenFunction.cpp

```diff
@@ -882,6 +883,11 @@ void SILGenFunction::emitGeneratorFunction(SILDeclRef function, VarDecl *var) {
   if (auto originalProperty = var->getOriginalWrappedProperty()) {
     if (originalProperty->isPropertyMemberwiseInitializedWithWrappedType()) {
       interfaceType = originalProperty->getPropertyWrapperInitValueInterfaceType();
+
+      if (auto fnType = interfaceType->getAs<AnyFunctionType>()) {
```

Comment:
Since this appears in two places in this PR already it might be worth extracting a new AnyFunctionType::withNoEscape(bool) and doing a quick pass over the callers of 'ExtInfo::withNoEscape()' to see if there are any other potential usages.